src/app-gocardless/banks/spk_worms_alzey_ried_malade51wor.js

```diff
@@ -0,0 +1,63 @@
+import Fallback from './integration-bank.js';
+
+import { printIban, amountToInteger } from '../utils.js';
+import { formatPayeeName } from '../../util/payee-name.js';
+
+/** @type {import('./bank.interface.js').IBank} */
+export default {
+  ...Fallback,
+
+  institutionIds: ['SPK_WORMS_ALZEY_RIED_MALADE51WOR'],
+
+  accessValidForDays: 90,
+
+  normalizeAccount(account) {
+    return {
+      account_id: account.id,
+      institution: account.institution,
+      mask: account.iban.slice(-4),
+      iban: account.iban,
+      name: [account.name, printIban(account)].join(' '),
+      official_name: account.product,
+      type: 'checking',
+    };
+  },
```

Comment:
Does this function require changing from the standard integration bank?

Reply:
No, I don't think so. Should I remove that function and calculateStartingBalance? I did not make changes to those.

Reply:
Yes please if they were not changed

Reply:
Both functions have been removed.